substrate/client/rpc-spec-v2/src/chain_head/chain_head.rs

```diff
@@ -575,30 +558,12 @@ where
 
 	async fn chain_head_unstable_continue(
 		&self,
-		ext: &Extensions,
-		follow_subscription: String,
-		operation_id: String,
+		_ext: &Extensions,
+		_follow_subscription: String,
+		_operation_id: String,
 	) -> Result<(), ChainHeadRpcError> {
-		let conn_id = ext
-			.get::<ConnectionId>()
-			.copied()
-			.expect("ConnectionId is always set by jsonrpsee; qed");
-
-		if !self.subscriptions.contains_subscription(conn_id, &follow_subscription) {
-			return Ok(())
-		}
-
-		let Some(operation) = self.subscriptions.get_operation(&follow_subscription, &operation_id)
-		else {
-			return Ok(())
-		};
-
-		if !operation.submit_continue() {
-			// Continue called without generating a `WaitingForContinue` event.
-			Err(ChainHeadRpcError::InvalidContinue.into())
-		} else {
-			Ok(())
-		}
+		// `WaitingForContinue`` is never emitted by the server.
+		Ok(())
```

Comment:
nit: Since we don't emit the pagination event anymore and the user is not required to call this method, does it still make sense to:
- keep the first part of the function which checks the subscription ID and operation IDs
- if they are valid, instead of `if !operation.submit_continue()` return directly `Err(ChainHeadRpcError::InvalidContinue.into())`?
This way we makes sure to follow the spec in those situations

Reply:
yeah, thanks missed that.

Basically, I'll just revert this change